solution/app/src/main/AndroidManifest.xml

```diff
@@ -29,6 +28,16 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/Theme.PlacesDemo">
+        <meta-data
+            android:name="com.google.android.gms.version"
+            android:value="@integer/google_play_services_version" />
+        <!--
+             The API key for Google Maps-based APIs.
```

Comment:
Maps SDK for Android

(this key in the Manifest is not used for the Places SDK)

Reply:
Changed as requested.